crates/astria-conductor/Cargo.toml

```diff
@@ -16,12 +16,12 @@ futures = { workspace = true }
 hex = { workspace = true }
 prost = { workspace = true }
 prost-types = { workspace = true }
-rand = { workspace = true }
-reqwest = { workspace = true, features = ["json"] }
 serde = { workspace = true }
 serde_json = { workspace = true }
 sha2 = { workspace = true }
+telemetry = { package = "astria-telemetry", path = "../astria-telemetry" }
```

Comment:
nit but why move this?

Reply:
didn't realize I moved it, it's bee put back